.github/workflows/release.yml

```diff
@@ -1,8 +1,11 @@
 name: Build and Release
 
-on:
-  release:
-    types: [created]
+on: push
+#  release:
+#    types: [created]
```

Comment:
Do we want a new release on every push on every branch? Imo this job should only run on when a release is created manually or when something is merged to `main`.

Reply:
this is temporary - removing now